ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -19,7 +19,9 @@ def initialize(opts = {})
       @servername       = opts.fetch(:servername, nil)
       @proxy_server     = opts.fetch(:proxy_server, @servername)
       @port             = opts.fetch(:port, @ssl ? "443" : "80")
-      @logroot          = opts.fetch(:logroot, "logs")
+      @accesslog        = opts.fetch(:accesslog, "logs/#{@servername ? @servername+'_access' : 'access'}#{@ssl ? '_ssl.log' : '.log'}")
```

Comment:
These are sort of hard to read.  Might be good to do a function like this:

```ruby
def default_accesslog
  prefix = @servername ? "#{@servername}_access" : 'access'
  suffix = @ssl ? '_ssl.log' : '.log'
  "logs/#{prefix}#{suffix}"
end
```

Same for error log.

Reply:
I've made a helper function for both cases. Please let me know what you think.

Reply:
Looks good. I committed the suggested change for that function to use a guard clause and return early.